src/styles/button.scss

```diff
@@ -0,0 +1,19 @@
+@import 'variables';
+
+.button__buy {
+  width: 166px;
+  height: 40px;
+  border: 1px solid $color__button;
+  border-radius: 4px;
+  background-color: $color__button;
+  color: $color__button__text;
+  font-weight: $font-weight-bold;
+  font-size: 14px;
+  outline: none;
+}
+
+.button__buy:hover {
+  background-color: $color__button-border;
+  color: $color__button;
+  border: 1px solid $color__button;
+}
```

Comment:
This button should be an element of card block, so move it back to card file and to card block